### PR TITLE
PERF: Move `EmailLog#reply_key` into new `post_reply_keys` table.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: spec/components/email/receiver_spec.rb [hunk 1]
```diff
@@ -813,13 +820,16 @@ def process(email_name)
     end
 
     context "with a valid reply" do
-      it "returns the destination when the key is valid" do
-        Fabricate(:email_log, reply_key: '4f97315cc828096c9cb34c6f1a0d6fe8')
+      it "returns the destyination when the key is valid" do
```
Comment (gschlager) [inline spec/components/email/receiver_spec.rb]: typo in `destyination` 😉

File: lib/email/sender.rb [hunk 2]
```diff
@@ -189,7 +195,7 @@ def send
         return skip(e.message)
       end
 
-      # Save and return the email log
+      # Save and return the email logtox
```
Comment (gschlager) [inline lib/email/sender.rb]: What's `logtox`?

Comment (tgxworld) [reply lib/email/sender.rb]: oops typo

File: spec/components/email/receiver_spec.rb (hunk 1 above)
Comment (tgxworld) [reply spec/components/email/receiver_spec.rb]: hehehe

**Pull request merged**